src/planners/rbt_star/RGBMTStar.cpp

```diff
@@ -177,7 +181,7 @@ bool planning::rbt_star::RGBMTStar::solve()
 
 std::shared_ptr<base::State> planning::rbt_star::RGBMTStar::getRandomState()
 {
-    int tree_idx;
+    size_t tree_idx;
```

Comment:
Initialization.

Reply:
On the other note, do we have a situation where tree search return no result?

Reply:
No, we don't, but I added `return nullptr` just in case.